src/integrations/admin/indexation-integration.php

```diff
@@ -169,6 +180,8 @@ public function render_indexation_warning() {
 	/**
 	 * Renders the indexation modal.
 	 *
+	 * @codeCoverageIgnore It echos the indexation modal.
```

Comment:
Please write test for this. You can test this by using `expectOutput` or `expectOutputContains`